nipype/interfaces/afni/preprocess.py

```diff
@@ -2146,11 +2146,12 @@ class SkullStrip(AFNICommand):
 
     def __init__(self, **inputs):
         super(SkullStrip, self).__init__(**inputs)
+
         if not no_afni():
             v = Info.version()
 
-            # As of AFNI 16.0.00, redirect_x is not needed
-            if v[0] > 2015:
+            # Between AFNI 16.0.00 and 16.2.07, redirect_x is not needed
+            if v >= (2016, 0, 0) and v < (2016, 2, 7):
```

Comment:
woah  - is this really true - crazy!

Reply:
oh yeah, tell me about it. I'll report this to AFNI, to make sure this is not a mistake linking.